.eslintrc.json

```diff
@@ -9,6 +9,9 @@
           "config/*.js"
         ]
       }
+    ],
+    "class-methods-use-this": [
```

Comment:
ESLint was complaining about the renderBody function of AssetsPage/AssetsTable/index.jsx, so we've chosen to ignore it here, as we find it to be an unhelpful case.